src/app/fdctl/run/tiles/quic.seccomppolicy

```diff
@@ -22,20 +22,7 @@ write: (or (eq (arg 0) 2)
 # arg 0 is the file descriptor to fsync.
 fsync: (eq (arg 0) logfile_fd)
 
-# openssl: RAND_bytes requires getpid
-#
-# QUIC uses OpenSSL RAND_bytes to generate randomness, which it uses for
-# picking connection IDs.  The OpenSSL implementation calls getpid() as
-# an implementation detail, they save the PID in a global and reseed the
-# RNG if the PID ever changes (the process was forked).  We don't need
-# this logic since our process can't fork.
-getpid
-
-# openssl: RAND_bytes requires getrandom
-#
-# QUIC uses OpenSSL RAND_bytes to generate randomness, which it uses for
-# picking connection IDs.  The OpenSSL implementation calls getrandom
-# internally for periodically reseeding the RNG.
+# QUIC uses getrandom for cryptographically secure randomness.
```

Comment:
Can you add a bit about why? It's useful as a reviewer and for auditors. I just spent an hour tracking the three consumers down and it might be useful to remove them for performance (syscall overhead) and in the case of #2 being misimplemented anyway. #1 can probably just be a PRNG. 

```suggestion
# quic: randomness is required for the protocol
#
# We generate random connection IDs for QUIC although the server
# can pick arbitrary IDs, we use random ones to prevent session linkage
# by adversaries across connection migration.
#
# QUIC retry packets use getrandom when creating the challenge token
# for the client to echo back. The implementation is questionable and
# probably the scheme can be improved.
#
# TLS ServerHello response requires a random number for key
# negotiation.
```